client/extensions/auto-translator/src/widget/translation-dialog/form.tsx

```diff
@@ -338,7 +342,9 @@ export const TranslationForm = () => {
         <FormSelect name="writethru" label="Writethru/Version">
           {getObjectKeys(values.translations).map((versionId) => (
             <option value={versionId} key={`version${versionId}`}>
-              {`Version ${capitalize(versionId)}`}
+              {`${currentVersion}` === versionId
+                ? `${capitalize(versionId)} (Current)`
```

Comment:
Can be a tad bit easier to read
`{capitalize(versionId) + (versionId === currentVersion ? " (Current)" : "")}`

Reply:
Current way is clear about what the label will be/what condition it's displayed. Concatenating is less flexible/can lead to unintended behaviour.